notebooks/demo2/train_relevance.ipynb

```diff
@@ -0,0 +1,5502 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.        s3_key="relevance_fine_tune_demo_scores,csv",
</pre><p>should this be <code>.csv</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/os-climate/aicoe-osc-demo/pull/155/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Reply:
Ah yes, it should be :)

Reply:
Updated!